packages/ui-position/src/Position/index.tsx

```diff
@@ -243,44 +172,43 @@ class Position extends Component<Props> {
   }
 
   componentWillUnmount() {
-    // @ts-expect-error ts-migrate(2339) FIXME: Property 'cancel' does not exist on type '() => vo... Remove this comment to see the full error message
+    // @ts-expect-error TODO: figure out if this is valid
     this.position.cancel()
```

Comment:
This is valid code (but pretty weirdly written), Ive checked in the debugger.
`position` is a medhod first (the one that calls `setState`), but in the constructor this variable is reassigned to a function returned by `debounce`, and this one has a `cancel` method. Its more clear in the compiled ES6 code what is happening:

```
  constructor(props) {
    ...
    this.position = () => {
      this.setState({
        positioned: true,
        ...this.calculatePosition(this.props)
      });
    };
    this.position = debounce(this.position, 0, {
      leading: false,
      trailing: true
    });
  }
```

IMO we should refactor this to get rid of this confusing code (its OK to do it in another ticket)

Reply:
For now I made a `Debounced` type that I could use here.